pkg/ansible/controller/reconcile.go

```diff
@@ -131,6 +137,8 @@ func (r *AnsibleOperatorReconciler) Reconcile(request reconcile.Request) (reconc
 
 	kc, err := kubeconfig.Create(ownerRef, "http://localhost:8888", u.GetNamespace())
 	if err != nil {
+		r.markError(u, request.NamespacedName, "Unable to run reconcilation")
+		logger.Error(err, "Unable to generate kubeconfig")
```

Comment:
should the logger.Error mention `Unable to run reconciliation`? seems weird the messages are different.

TYPO: reconcilation -> reconciliation

Reply:
One is a user-facing error, one is for an admin to debug the operator. I don't think the user cares about the kubeconfig, the admin will.